src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -96,29 +98,23 @@ class Favourites extends Component {
         this.props.bindings.toggleFavourite(stockCode);
     }
 
-    addDropTarget(id) {
-        const dropTarget = document.getElementById(id);
-        this.dropTarget = dropTarget;
-
-        dropTarget.addEventListener('drop', e => {
-            const codes = this.props.favourites.codes;
-            const code = e.dataTransfer.getData('text/plain');
-            this.props.dispatch(insertFavouriteAt(codes.length, code));
-            dropTarget.classList.remove('dragOver');
-        }, false);
-
-        dropTarget.addEventListener('dragover', e => {
-            if (!dropTarget.classList.contains('dragOver')) {
-                dropTarget.classList.add('dragOver');
-            }
-            e.preventDefault();
-        }, false);
+    onDrop(e) {
+        const codes = this.props.favourites.codes;
+        const code = e.dataTransfer.getData('text/plain');
+        this.props.dispatch(insertFavouriteAt(codes.length, code));
+        e.target.classList.remove('dragOver');
```

Comment:
[`e.target`](https://developer.mozilla.org/en-US/docs/Web/API/Event/target) is not equivalent to the `dropTarget` in the previous implementation - I'm not sure if this is the intention? I think that `this` ([in this case: `this === e.currentTarget`](https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget)) would be more appropriate and consistent here, unless there is a reason to change?

This [StackOverflow answer](http://stackoverflow.com/a/21667010) explains the difference between `event.target` and `this`/`event.currentTarget` well.